config/environments/development.rb

```diff
@@ -11,7 +11,7 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = false
+  config.eager_load = true
```

Comment:
The comment was just _asking_ for this to happen

```
# Eager load code on boot?
```

or delete imo

Reply:
Ah, that was just for order-dependent loading issues debugging :D Definitely deleting